h2o-core/src/main/java/hex/Model.java

```diff
@@ -117,7 +117,15 @@ public void setInputParms(P _input_parms) {
   }
 
   public interface UpdateAuxTreeWeights {
-    void updateAuxTreeWeights(Frame frame, String weightsColumn);
+    UpdateAuxTreeWeightsReport updateAuxTreeWeights(Frame frame, String weightsColumn);
+
+    class UpdateAuxTreeWeightsReport {
+      public int[] _warn_trees;
```

Comment:
Also camelCase

Reply:
this is the better way for public-facing classes